examples/client/client.c

```diff
@@ -1099,455 +1099,674 @@ static int ClientRead(WOLFSSL* ssl, char* reply, int replyLen, int mustRead,
+        {
+            OPTION_TYPE_USE_X448, '8',
+            "-8          Use X448 for key exchange\n",
+            "-8          鍵交換に X448 を使用する\n", 
+    },
 #endif
 #if defined(OPENSSL_ALL) && defined(WOLFSSL_CERT_GEN) && \
     (defined(WOLFSSL_CERT_REQ) || defined(WOLFSSL_CERT_EXT)) && \
     !defined(NO_FILESYSTEM) && !defined(NO_WOLFSSL_DIR)
-        "-9          Use hash dir look up for certificate loading\n"
-        "            loading from <wolfSSL home>/certs folder\n"
-        "            files in the folder would have the form \"hash.N\" file name\n"
-        "            e.g symbolic link to the file at certs folder\n"
-        "            ln -s ca-cert.pem  `openssl x509 -in ca-cert.pem -hash -noout`.0\n",
-                                                                        /* 67 */
+        {
+            OPTION_TYPE_USE_HASH_DIR, '9',
+            "-9          Use hash dir look up for certificate loading\n"
+            "            loading from <wolfSSL home>/certs folder\n"
+            "            files in the folder would have the form \"hash.N\" file name\n"
+            "            e.g symbolic link to the file at certs folder\n"
+            "            ln -s ca-cert.pem  `openssl x509 -in ca-cert.pem -hash -noout`.0\n",
+            "-9          証明書の読み込みに hash dir 機能を使用する\n"
+            "            <wolfSSL home>/certs フォルダーからロードします\n"
+            "            フォルダー中のファイルは、\"hash.N\"[N:0-9]名である必要があります\n"
+            "            以下の例ではca-cert.pemにシンボリックリンクを設定します\n"
+            "            ln -s ca-cert.pem  `openssl x509 -in ca-cert.pem -hash -noout`.0\n",
+    },
 #endif
 #if defined(WOLFSSL_WOLFSENTRY_HOOKS) && !defined(NO_FILESYSTEM) && \
     !defined(WOLFSENTRY_NO_JSON)
-        "--wolfsentry-config <file>    Path for JSON wolfSentry config\n",
-                                                                        /* 68 */
+        {
+            OPTION_TYPE_WOLFSENTRY_CONFIG_FILE, '--wolfsentry-config',
+            "--wolfsentry-config <file>    Path for JSON wolfSentry config\n",
+            "--wolfsentry-config <file>    wolfSentry コンフィグファイル\n",
+    },
 #endif
 #ifndef WOLFSSL_TLS13
-        "-7          Set minimum downgrade protocol version [0-3] "
-        " SSLv3(0) - TLS1.2(3)\n",
+        {
+            OPTION_TYPE_SET_MIN_DOWNGRADE_PROTOCOL_VER, '7',
+            "-7          Set minimum downgrade protocol version [0-3] "
+            " SSLv3(0) - TLS1.2(3)\n",
+            "-7          最小ダウングレード可能なプロトコルバージョンを設定します [0-3] "
+            " SSLv3(0) - TLS1.2(3)\n",
+
+    },
 #else
-        "-7          Set minimum downgrade protocol version [0-4] "
-           " SSLv3(0) - TLS1.3(4)\n",                                   /* 69 */
+        {
+            OPTION_TYPE_SET_MIN_DOWNGRADE_PROTOCOL_VER, '7',
+            "-7          Set minimum downgrade protocol version [0-4] "
+           " SSLv3(0) - TLS1.3(4)\n",
+            "-7          最小ダウングレード可能なプロトコルバージョンを設定します [0-4] "
```

Comment:
Please keep the line under our 80-character line length limit per our Coding Standard